src/NuGet/Microsoft.Net.Compilers.Toolset/AnyCpu/Microsoft.Net.Compilers.Toolset.Package.csproj

```
@@ -47,7 +47,7 @@
     <Content Include="buildMultiTargeting\**\*.*" PackagePath="buildMultiTargeting\" />
   </ItemGroup>
 
-  <Target Name="_GetFiles" DependsOnTargets="$(_InitializeArtifactsTargetName);Build">
+  <Target Name="GetFiles" DependsOnTargets="$(_InitializeArtifactsTargetName);Build">
```

```suggestion
  <Target Name="ComputeFilesAndCopy" DependsOnTargets="$(_InitializeArtifactsTargetName);Build">
```

If we don't take the other suggestion just to move the copy to a separate target.